.golangci-warnings.yml

```diff
@@ -0,0 +1,76 @@
+run:
+  timeout: 5m
+  tests: true
+  
+linters:
+  disable-all: true
+  enable:
+    - staticcheck
+    - structcheck
+    - typecheck
+    - varcheck
+    - deadcode
+    - gosimple
+    - unused
+    - partitiontest
+
+
+linters-settings:
+  custom:
+    partitiontest:
+      path: cmd/partitiontest_linter/plugin.so
+      description: This custom linter checks files that end in '_test.go', specifically functions that start with 'Test' and have testing argument, for a line 'partitiontest.ParitionTest(<testing arg>)'
+      original-url: github.com/algorand/go-algorand/cmd/partitiontest_linter
+
+severity:
+  default-severity: warning
+
+issues:
+  # use these new lint checks on code since #2574
+  new-from-rev: eb019291beed556ec6ac1ceb4a15114ce4df0c57
```

Comment:
What happens when we add new linters, will we just update this line to the commit we add new linters?

Reply:
yes, or make an effort to "clean" out the repo for that specific linter like I did in https://github.com/algorand/go-algorand/pull/2574

Reply:
yes? @cce what do you think?